concordia/templates/home.html

```diff
@@ -51,49 +51,48 @@ <h4 class="offwhite-text mt-half">REVIEW</h4>
+                        {% if 'Lincoln' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/AbrahamLincoln.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Rosa Parks' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/rosa-parks-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Clara Barton' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/clara-barton-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Branch Rickey' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/branch-rickey-campaign.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Mary Church Terrell' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/MaryChurchTerrell.svg' %}" alt="{{ campaign.title }} image">
+                        {% elif 'Bourne' in campaign.title %}
+                        <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/WilliamOlandBourne.svg' %}" alt="{{ campaign.title }} image">
+                        {% else %}
+                        {% comment %} FIXME: when we implement campaign media, we also want a simple way to get the MEDIA_URL-relative thumbnail {% endcomment %}
+                        <img class="img-fluid" alt="{{ campaign.title }} image" src="{% if campaign.s3_storage %}{{ campaign.media_url }}{% else %}{{ MEDIA_URL }}{{ campaign.media_url }}{% endif %}">
+                        {% endif %}
+
+                        <h3 class="mt-half">{{ campaign.title }}</h3>
+                </div>
+                {% endfor %}
             </div>
-            <div class="col-12 col-md-4 mb-default text-center border-left-home-contribute">
-                <img class="img-home-contribute d-block mx-auto img-fluid" src="{% static 'img/WilliamOlandBourne.svg' %}"
-                    alt="William Oland Bourne Papers">
-                <h3 class="mt-half">William Oland Bourne Papers</h3>
-                <a class="btn btn-outline-primary" href="#" role="button">Get Started!</a>
+            <div class="col mx-auto d-flex justify-content-center my-default">
+                <a class="btn btn-primary" href="{% url 'transcriptions:campaigns' %}" role="button">Browse All Campaigns</a>
             </div>
         </div>
-        <div class="col mx-auto d-flex justify-content-center my-default">
-            <a class="btn btn-primary" href="{% url 'transcriptions:campaigns' %}" role="button">Browse More Campaigns</a>
-        </div>
     </div>
-</div>
-{% endblock main_content %}
+    {% endblock main_content %}
```

Comment:
whitespace

Reply:
Is there a VS Code / prettier setting for django template language? I noticed your indentations are different than mine.

Reply:
Prettier is automatically indenting that when I save the file. :(

Reply:
I've been using https://github.com/vscode-django/vscode-django but that doesn't format at all. I've been okay with that since many things don't handle the indentation of HTML + Django template tags well.

Re: the indentation, there was a missing close tag which is why Prettier was adding the extra indent